.github/renovate.json

```
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "constraints": {"python": "3.10.6"},
+  "extends": ["config:base"],
+  "enabledManagers": ["ansible-galaxy", "gradle", "pip-compile"],
+  "pip-compile": {"fileMatch": ["^requirements(-dev)?\\.in$"]}
```

```suggestion
  "pip-compile": {"fileMatch": ["^requirements-dev\\.in$"]}
```
since we have a single file.